projects/example-app/src/app/books/components/book-search.component.ts

```diff
@@ -11,7 +11,7 @@ import { Component, Output, Input, EventEmitter } from '@angular/core';
             matInput
             placeholder="Search for a book"
             [value]="query"
-            (keyup)="search.emit($event.target.value)"
+            (keyup)="search.emit($any($event.target).value)"
```

Comment:
```suggestion
            (keyup)="onSearch($event)"
```

Add `onSearch` method:

```ts
  onSearch(event: KeyboardEvent): void {
    this.search.emit((event.target as HTMLInputElement).value);
  }
```